views/default/page/layouts/widgets.php

```diff
@@ -15,10 +15,14 @@
 $show_access = elgg_extract('show_access', $vars, true);
 
 $owner = elgg_get_page_owner_entity();
+$context = elgg_get_context();
```

Comment:
in the hook you can always use elgg_get_context_stack or elgg_in_context, so you should be good